pkg/lhutil/lighthouse.go

```diff
@@ -102,6 +115,11 @@ func GetServiceExportCondition(status *mcsv1a1.ServiceExportStatus, ct mcsv1a1.S
 	return latestCond
 }
 
+func ServiceExportConditionEqual(c1, c2 *mcsv1a1.ServiceExportCondition) bool {
+	return c1.Type == c2.Type && c1.Status == c2.Status && reflect.DeepEqual(c1.Reason, c2.Reason) &&
+		reflect.DeepEqual(c1.Message, c2.Message)
```

Comment:
`Message` and `Reason` are strings so `==` works? 
If so, no need to call DeepEqual. DeepEqual is for structs.

Reply:
Got this function directly from the agent code, wasn't sure if I was missing something.
Changed it.